convert/convert.go

```diff
@@ -210,3 +223,56 @@ func getWigChromIndex(s string, wigSlice []*wig.Wig) int {
 	log.Fatalf("Bed Chromosome, %s, not in reference genome.", s)
 	return -1
 }
+
+//PairwiseFaToVcf takes in a pairwise multiFa alignment and returns Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
+//This will have to be done by chromosome, as a pairwise multiFa will only have two entries, thus containing one chromosome per file.
+func PairwiseFaToVcf(f []*fasta.Fasta, chr string) []*vcf.Vcf {
+	var pastStart bool = false //bool check to see if we have an insertion at the start of an alignment.
+	var insertion bool = false
+	var deletion bool = false
+	var insertionAlnPos int
+	var deletionAlnPos int
+	answer := make([]*vcf.Vcf, 0)
+
+	for i := 0; i < len(f[0].Seq); i++ { //loop through reference alignment positions
+		if f[0].Seq[i] == dna.Gap { //reference is gap (insertion)
+			if pastStart {
+				if !insertion {
+					insertionAlnPos = i - 1 //TODO: breaks for alignments starting with an insertion(gap in Aln postion 0 of the reference)
```

Comment:
What happens now when this is in the input?  Can we print an error message and die?

Reply:
Oh I should remove this comment, the pastStart was an addition in the latest commit that solves this problem